Example/ViewController.swift

```diff
@@ -54,6 +56,7 @@ class ViewController: UIViewController {
             displayController?.miniAppDisplay = miniAppDisplay
             self.currentMiniAppInfo = nil
             self.currentMiniAppView = nil
+            self.displayController = displayController
```

Comment:
we could use the `self.presentedViewController as? DisplayNavigationController` when needed instead of creating a `displayController` variable. Do you think it could be relevant here?

Reply:
Oh sorry. This object is not used anymore. My bad, I removed this one now.